docs/thrift.md

```diff
@@ -208,3 +208,47 @@ message with error code (`code:1`) `0x05` (unexpected error).
   [`arg1`]: #arg1
   [`arg3`]: #arg3
 
+
+Streaming Thrift
+================
+
+Streaming Thrift allows requests or responses to be streamed over TChannel.
+
+Streaming Thrift will uses a separate `as` header: `sthrift`, and is based
+on the standard Thrift protocol, with the only exception being the encoding
+of arg3 in both the `call req` and `call res`.
+
+The encoding of data for streaming arg3 is a 4 byte length-prefixed chunk:
+```
+chunk~4 chunk~4 chunk~4 ...
```

Comment:
each chunk is a struct. so if you have a `method(1: string foo, 2: string bar, 3: string baz)`

Then you should expect each chunk to be `{ foo: ..., bar: ..., baz: ... }`